app/layout.tsx

```diff
@@ -15,17 +16,20 @@ export const metadata: Metadata = {
     },
 };
 
-export default function RootLayout({children}: Readonly<{ children: React.ReactNode }>) {
+export default function RootLayout({ children }: Readonly<{ children: React.ReactNode }>) {
     return (
         <html lang="en">
-        <body
-            className={`${montserrat.className}   h-full w-full bg-white text-white`}>
-        <Navigation/>
-        <main>
-            <Suspense>{children}</Suspense>
-        </main>
-        <Footer/>
-        </body>
+            <body className={`${montserrat.className} h-full w-full bg-white text-white`}>
+                <RecaptchaProvider>
```

Comment:
Like I mentioned in the `recaptcha.tsx` component, you don't need this wrapper, just use the one provided by the library.